pkg/apis/kops/instancegroup.go

```diff
@@ -76,6 +76,8 @@ type InstanceGroupSpec struct {
 	RootVolumeSize *int32 `json:"rootVolumeSize,omitempty"`
 	// RootVolumeType is the type of the EBS root volume to use (e.g. gp2)
 	RootVolumeType *string `json:"rootVolumeType,omitempty"`
+	// RootVolumeOptimization enables EBS optimization for an instance
+	RootVolumeOptimization *bool `json:"rootVolumeOptimization,omitempty"`
```

Comment:
Agreed - we can put a TODO in there to rationalize in the next API version.  I think also now that local volumes are coming, people will soon be asking for non-root EBS attachments also.